src/main/java/sirius/biz/translations/MultiLanguageString.java

```diff
@@ -323,6 +323,11 @@ public boolean isWithFallback() {
 
     @Override
     public void setData(Map<String, String> newData) {
+        if (newData == null) {
+            super.setData(Collections.emptyMap());
```

Comment:
shouldn't we simply clear our map here - Collections.emptyMap() returns an unmodifyable map(?) which would lead to successive failures...

Reply:
Good point 👍  Didn't consider that the emptyMap is unmodifiable